### PR TITLE
add validation error messages

### PR DESCRIPTION
**Pull request by cmacaulay**
🎉 
- Error message appear when name or calories fields are not filled out
- Error message appears when they are both not filled out
- Error messages clear when new food is successfully submitted
- Error messages match what Pivotal Tracker Cards ask for 
- Error message will clear if not longer applicable (e.g. if both were empty, and a name has been entered)
- Some minor refactoring to previous code
- Updated adding a column to a table from append to prepend, so as to follow Pivotal Tracker card "Creating A Food" id#144889853
- Minor styling to the error messages (see below)
screen shot 2017-05-13 at 6 25 27 pm

### CODE REVIEW

File: foods.html [hunk 1]
```diff
@@ -17,10 +17,14 @@ <h1 class="center">Manage Foods</h1>
       <div class="column column-20">
         <form class="new-food">
           <fieldset>
-            <label class="center" for="name-field">Name</label>
+            <label id="name-field" class="center" for="name-field">Name</label>
             <input type="text" placeholder="Pistachios" name="food-name">
-            <label class="center" for="colorie-amount">Calorie Amount</label>
+            <div class="error name-validation-error"></div>
```
Comment (TheKaterTot) [inline foods.html:22]: I like how clear the names of the css selectors are

File: lib/foods.js [hunk 2]
```diff
@@ -33,7 +37,30 @@ function addRow(food) {
         <td class='food-calories'>${food.calories}</td>
         <td class='food-delete'><i class="material-icons">remove_circle</i></td>
       </tr>`;
-  $(".food-options tbody").append(foodRow);
+  $(".food-options tbody").prepend(foodRow);
+};
+
+function handleFieldErrors (newFood) {
+  clearErrors();
```
Comment (TheKaterTot) [inline lib/foods.js:44]: I like very much how you've broken this code down into smaller functions

**Comment by TheKaterTot** [comment]
Very nice! Thanks for the screenshots, too. 💛 🎆

File: lib/foods.js (hunk 2 above)
Comment (cmacaulay) [reply lib/foods.js:44]: Yay! I thought you might :)

**Pull request merged**